sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/csv/CSVRelation.scala

```diff
@@ -227,6 +227,12 @@ private[csv] class CsvOutputWriter(
   }
 
   private def makeConverter(dataType: DataType): ValueConverter = dataType match {
+    case ByteType | ShortType | IntegerType | LongType =>
+      (row: InternalRow, ordinal: Int) => row.get(ordinal, dataType).toString
+
+    case FloatType | DoubleType | _: DecimalType | BooleanType | StringType =>
+      (row: InternalRow, ordinal: Int) => row.get(ordinal, dataType).toString
```

Comment:
What is the difference with the case and the case above? The result is the same right?